ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/Errors.xml

```diff
@@ -188,13 +195,14 @@
     <QuickFix>ReplaceLambdaWithInnerExpressionFix</QuickFix>
   </Warning>
 
-  <Warning staticGroup="FSharpErrors" name="ExpressionCanBeReplacedWithId">
-    <Parameter type="IFSharpExpression" name="expr"/>
-    <Range>expr.GetHighlightingRange()</Range>
+  <Warning staticGroup="FSharpErrors" name="LambdaCanBeReplacedWithOperator">
```

Comment:
Please replace `Operator` with `Reference` or `BuiltinFunction`. None of the current cases where this warning is produced is about operators.